rbac_api_with_domains.go

```diff
@@ -42,3 +42,19 @@ func (e *Enforcer) AddRoleForUserInDomain(user string, role string, domain strin
 func (e *Enforcer) DeleteRoleForUserInDomain(user string, role string, domain string) (bool, error) {
 	return e.RemoveGroupingPolicy(user, role, domain)
 }
+
+// DeleteRolesForUserInDomain deletes all roles for a user inside a domain.
+// Returns false if the user does not have any roles (aka not affected).
+func (e *Enforcer) DeleteRolesForUserInDomain(user string, domain string) (bool, error) {
```

Comment:
Using `e.RemoveFilteredGroupingPolicy` is easier.

Reply:
`e.RemoveFilteredGroupingPolicy` can only filter one field, how to find `domain` and `user` at the same time?

Reply:
okay, you are right.